### PR TITLE
Update for Laravel 10/11

### PR DESCRIPTION
**Pull request by askmrsinh**
Summary of changes:
- Changes package versions to support Laravel 10/11, drops older Laravel versions
- Updates PHPUnit to version 10
- Updates minimum PHP requirement to 8.1, drops older versions
- Updates Github Workflow to use newer action packages

Addresses https://github.com/laravel-notification-channels/skeleton/issues/32.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,13 +12,13 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
-        "illuminate/notifications": "~6.0 || ~7.0 || ~8.0",
-        "illuminate/support": "~6.0 || ~7.0 || ~8.0"
+        "php": ">=8.1",
+        "illuminate/notifications": "~10.0",
```
Comment (atymic) [inline composer.json:16]: May as well add 11 here as it will be out soon.

**Review by atymic — APPROVED** [review]

**Comment by atymic** [comment]
One minor change :)

File: composer.json (hunk 1 above)
Comment (askmrsinh) [reply composer.json]: Done!

**Comment by Parables** [comment]
if I may ask, why is this still opened?

**Pull request merged**